contrib/signet/getcoins.py

```diff
@@ -49,4 +49,16 @@
 except:
     print('Unexpected error when contacting faucet:', sys.exc_info()[0])
     exit()
-print(res.text)
+
+# Display the output as per the returned status code
+if res:
+    # When the return code in between 200 and 400 i.e. successful
+    print(res.text)
+elif res.status_code == 400:
+    print('You can use the global faucet only once per 24hrs. Please try again later.')
```

Comment:
```suggestion
webbrowser.open('https://signetfaucet.com/', new = 1)
```

Reply:
`getcoins.py` is a script for a reason. You can't assume that every machine has a GUI or even a browser installed. Especially if this script is used in some automated way.